shared/settings/notifications/container.tsx

```diff
@@ -34,11 +34,13 @@ export default Container.connect(
     _groups: state.settings.notifications.groups,
     allowEdit: state.settings.notifications.allowEdit,
     mobileHasPermissions: state.push.hasPermissions,
+    showEmailSection: !state.settings.email.emails,
```

Comment:
Should this check `state.settings.email.emails.size` too?

Reply:
Oops, also I removed an `!` in debugging and forgot to put it back.